includes/Modules/Analytics.php

```diff
@@ -1404,11 +1411,25 @@ public function handle_token_response_data( $token_response ) {
 	 * @return array Updated array with the mode=analytics-step parameter.
 	 */
 	private function update_proxy_setup_mode( $params ) {
-		if ( Feature_Flags::enabled( 'serviceSetupV2' ) && ! $this->is_connected() ) {
-			$params['mode'] = 'analytics-step';
+		$mode = $this->get_proxy_setup_mode();
+		if ( ! empty( $mode ) ) {
+			$params['mode'] = $mode;
 		}
 
 		return $params;
 	}
```

Comment:
See above, this method can then be removed.

Reply:
Removed.